db/keyval/filedb/README.md

```diff
@@ -0,0 +1,66 @@
+# FileDB - file system configuration reader plugin
+
+The fileDB plugin allows to use the file system of a operating system as a key-value data store. The filesystem
+plugin watches for pre-defined files or directories, reads a configuration and sends response events according
+to changes.
+
+All the configuration is resynced in the beginning (as for standard key-value data store). Configuration files
+then can be added, updated, moved, renamed or removed, plugin makes all the necessary changes.
+
+Important note: fileDB as datastore is read-only from the plugin perspective, changes from within the plugin
+are not allowed.
+
+## Configuration
+
+All files/directories used as a data store must be defined in configuration file. Location of the file
+can be defined either by the command line flag `filedb-config` or set via the `FILEDB_CONFIG`
+environment variable.
+
+## Data structure
+
+Plugin currently supports only JSON and YAML-formatted data. The format of the file is as follows for JSON:
```

Comment:
List supported formats using list to make it more obvious at first glance, something like:

### Supported formats:
* JSON (`*.json`)
* YAML (`*.yaml`)